microcosm_logging/factories.py

```diff
@@ -113,21 +114,22 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
-        "loggerName": "%(name)s",
-        "levelName": "%(levelname)s",
-        "message": "%(message)s",
-        # when was it logged?
-        "ascTime": "%(asctime)s",
-        # where in the code was it logged?
-        "fileName": "%(filename)s",
-        "functionName": "%(funcName)s",
-        "lineNo": "%(lineno)d",
-    }
+    fields = [
+        'asctime',
+        'levelname',
+        'name',
+        'filename',
+        'funcName',
+        'lineno',
+        'message',
+    ]
+
+    log_format = lambda x: ['%({0:s})'.format(i) for i in x]
```

Comment:
Better syntax:

```
formatter = "pythonjsonlogger.jsonlogger.JsonFormatter"
format = " ".join("{{{}}}:s".format(field) for field in fields)

return {
    "()": formatter,
    "format": 
}
```

Why?
 - Use newer style string formatting
 - Avoid the lambda
 - Parallel structure